COMMERCE-10054

brianchandotcom opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest(master)/4845/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-16.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4845/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 20 out of 23 jobs passed in 1 hour 18 minutes 56 seconds 699 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/746f6c2e563271d74686fe8940408061ab93b3cf">746f6c2e563271d74686fe8940408061ab93b3cf</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/8c2f1493536e8380342b0a7dc6d9298ae4c02442">8c2f1493536e8380342b0a7dc6d9298ae4c02442</a></p>20 out of 23 jobs PASSED<div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest(master)/4845/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/283843/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324855/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</strong></strike></a></li></ul></div><details><summary><strong>20 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/83540/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327754/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/283844/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/346839/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/346838/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324854/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/346840/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/83541/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326666/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324851/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/274093/">test-portal-acceptance-pullrequest-batch(master)/js-unit-private-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/274094/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324850/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324856/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/341359/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324849/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324852/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/327755/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/274095/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324853/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-16.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4845/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/283843/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></h5><div><h6>Job Results:</h6><p>14 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/283843//consoleText">AXIS_VARIABLE=0,label_exp=!master #283843</a><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/283843//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithSearch_SearchEditableText_">LocalFile.ContentPagesWithSearch#SearchEditableText</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-16/1570810226125/test-portal-acceptance-pullrequest(master)/4845/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.ContentPagesWithSearch_SearchEditableText/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-16/1570810226125/test-portal-acceptance-pullrequest(master)/4845/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.ContentPagesWithSearch_SearchEditableText/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-16/1570810226125/test-portal-acceptance-pullrequest(master)/4845/functional-tomcat90-mysql57-jdk8/1/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">acceptance upstream results</a> at 0a37c30806e3be96b1433866e1a235e61684bf37:</strong></summary><ol><li><div><h5><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/324855/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/324855//consoleText">AXIS_VARIABLE=0,label_exp=!master #324855</a><div><p>Please fix <strong>semantic versioning</strong> on <strong><a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-102798">katienesterovich/LPS-102798</a></strong><pre><code>     [exec] &gt; Task :apps:layout:layout-taglib:syncVersions
     [exec] &gt; Task :apps:layout:layout-content-page-editor-web:compileJSP
     [exec] 
     [exec] &gt; Task :apps:layout:layout-admin-web:generateJSPJava
     [exec] Compiling JSP files of project ':apps:layout:layout-admin-web' with /opt/dev/projects/github/liferay-portal/bundles/tomcat-9.0.17/webapps/ROOT/WEB-INF/lib/util-taglib.jar as dependency in place of 'com.liferay.portal:com.liferay.util.taglib'
     [exec] 
     [exec] &gt; Task :apps:layout:layout-admin-web:compileJSP
     [exec] Note: Some input files use or override a deprecated API.
     [exec] Note: Recompile with -Xlint:deprecation for details.
     [exec] Note: Some input files use unchecked or unsafe operations.
     [exec] Note: Recompile with -Xlint:unchecked for details.
     [exec] 
     [exec] &gt; Task :apps:layout:layout-content-page-editor-web:autoUpdateXml SKIPPED
     [exec] &gt; Task :apps:layout:layout-content-page-editor-web:jar
     [exec] &gt; Task :apps:layout:layout-content-page-editor-web:baseline SKIPPED
     [exec] &gt; Task :apps:layout:layout-content-page-editor-web:syncVersions
     [exec] &gt; Task :apps:layout:layout-admin-web:autoUpdateXml SKIPPED
     [exec] 
     [exec] &gt; Task :apps:layout:layout-admin-web:jar
     [exec] warning: Unable to determine whether the annotation org.osgi.annotation.versioning.ProviderType applied to type com.liferay.exportimport.data.handler.base.BaseStagedModelDataHandler is a component property type as it is not on the project build path. If this annotation is a component property type then it must be present on the build path in order to be processed
     [exec] 
     [exec] &gt; Task :apps:layout:layout-admin-web:baseline SKIPPED
     [exec] &gt; Task :apps:layout:layout-admin-web:syncVersions
     [exec] 377 actionable tasks: 347 executed, 30 up-to-date
     [exec] Gradle build finished at 2019-10-11 16:59:30.574.
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:layout:layout-seo-api:baseline'.
     [exec] &gt; Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/tools/sdk/dist/com.liferay.layout.seo.api-2.0.0.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay/com.liferay.layout.seo.api/1.1.0/1c4f8555c1a63f1774d0e92a748aa619d23ce794/com.liferay.layout.seo.api-1.1.0.jar</code></pre></p></div></div></li></ol></div></li></ol></details></details></html>

brianikim commented:
reviewing now

brianikim commented:
Hello @katienesterovich ,

The decision to add CharPool.NUMBER_0 was introduced as part of LPS-96828 (https://github.com/liferay/liferay-portal/commit/d3ed9807d8579714b7b9fa58c104d41bc24b3f51#diff-7756c4655f18bd336723ab2b419c1556).

Please investigate whether your change needs to be implemented in the other files changed as part of that commit. Also, please investigate whether your change does not cause LPS-96828 again.

john-co committed:
LPS-169672 Adjust test path locator for field name change

Commit:
LPS-169672 Add client-extentions type description

Commit:
LPS-169672 Fix remote-apps repeatable fields visual bug

Commit:
LPS-169672 Fix style of client-extensions

Commit:
LPS-169672 Fix client-extension iframe fields order and labels

izaera committed:
LPS-169672 Rename getHelpNameLabel to getHelpLabel

izaera committed:
LPS-169672 Validate required name server side

Commit:
LPS-169672 Update getHelpLabel in client-extension

brianikim committed:
COMMERCE-10054 Do not provide guest-default resource permissions at all

Commit:
LPS-169672 Update properties text for client-extension

Commit:
LPS-169672 Add client-extensions texts

Commit:
LPS-169672 Minor fixes

Commit:
LPS-169672 Fix RemoteApp testcases

kristinonias committed:
LPS-156320 Add CanTriggerActionWithExpressionByAddingEntry test

Commit:
LPS-169672 Add the input HTML Element Name to RemoteAppsEntry.addEntry macro

Commit:
LPS-169672 Refactor tests about error message in RemoteApps testcase

Commit:
LPS-169672 SF

Commit:
LPS-169672 change  to {} in testcases

ealonso committed:
LPS-174989 Unused

dsanz committed:
LPS-169672 Use correct order

dsanz committed:
LPS-169672 Remove <liferay-frontend:fieldset-group> tag usage

dsanz committed:
LPS-169672 Fix jspc error and minor SF

brianchandotcom committed:
LPS-169672 This is already auto generated, type: git grep DuplicateClientExtensionEntryExternalReferenceCodeException **PersistenceImpl**

brianchandotcom committed:
LPS-169672 Use generated class because 'name' is a real column

brianchandotcom committed:
LPS-169672 regen

dsanz committed:
LPS-169672 Manage ClientExtensionEntryNameException in the front end

dsanz committed:
LPS-169672 not needed

dsanz committed:
LPS-169672 Rephrase UI messages

dsanz committed:
LPS-169672 Make friendly url help message consistently appear both in iframe and custom-element client extensions

dsanz committed:
LPS-169672 build lang

dsanz committed:
LPS-169672 sort

dsanz committed:
LPS-169672 Remove links from Language.properties. Reprhase and use lower case for custom element types

dsanz committed:
LPS-169672 not needed as CETLabelUtil already returns the translated value

dsanz committed:
LPS-169672 enable opt-in usage of <liferay-learn:message> tag for any client extension type, and provide help resources for existing learn documentation

dsanz committed:
LPS-169672 SF

brianikim committed:
COMMERCE-10054 delete unsupported resourcePermission

brianikim committed:
COMMERCE-10054 Sales Agent needs ADD_COMMERCE_ORDER permission

petershin committed:
LPS-166479 Publish everything

brianikim committed:
COMMERCE-10054 Update Sales Agent Role permission

ChrisKian committed:
LPS-172417 Remove PortalUtil.getPathProxy since PortalUtil.getPathContext() already accounts for the proxy path.

ChrisKian committed:
LPS-172417 If applicable, separate modulePortletId from name to ensure the proxy and context path are also removed from resource path if needed.

brianikim committed:
COMMERCE-10054 ADD_COMMERCE_ORDER permission check only if business account

julschong committed:
LPS-172713 Remove Dom4jUtil usage in WSDDMerger and use Java2WsddTask.documentToString instead

brianchandotcom committed:
LPS-169672 Wordsmith

brianchandotcom committed:
LPS-169672 Wordsmith

brianchandotcom committed:
LPS-169672 Wordsmith

brianchandotcom committed:
LPS-169672 Wordsmith? The language keys seems very inconsistent. What is an IFrame type?

dsanz committed:
LPS-169672 Wordsmith. Add theme svg help message.

dsanz committed:
LPS-169672 build lang

dsanz committed:
LPS-169672 SF

ChrisKian committed:
LPS-172417 Names which start with modulePortletIds should not be stripped of their context path, since we cannot add it back later like we do for non-module portlet names.

ChrisKian committed:
LPS-172417 SF, improve performance by concating string only when needed

brookedalton committed:
LPS-172705 Typo

michaelhashimoto committed:
LRCI-3404 Create a Spring Boot Application client-extension with basic configurations

michaelhashimoto committed:
LRCI-3404 Add simple endpoint for rest api

michaelhashimoto committed:
LRCI-3404 Change the default server port for the Spring Boot Server application

brianikim committed:
COMMERCE-10540 Disable AddToCart button if business account and no add permission.

jeffwu0724 committed:
LPS-173503 remove the usage of stream in IncludeExcludeUtil

brianikim committed:
COMMERCE-10054 update schemaVersion

Commit:
LPS-169672 Adjust some path names for a test on CI

Commit:
LPS-169672 SF

julschong committed:
LPS-172713 Use petra-string StringPool instead

julschong committed:
LPS-172713 Remove Dom4jUtil usage in Java2WsddTask and use dom4j library directly

julschong committed:
LPS-172713 Use newly added Java2WsddTask instead in WSDDBuilder

julschong committed:
LPS-172713 Use UnsyncStringReader instead of XMLSafeReader
The benefit of using XMLSafeReader is that any character besides '<' passed in before the prolog tag is removed before processing.
Current, there is no wsdd input xml file with preceding characters.
After this change, all wsdd input xml files cannot have preceding characters

Commit:
LPS-169672 Remove variables not needed in macro

georgel-pop-lr committed:
LPS-173575 Use flex class for unstyled to be displayed correctly on a Content page

georgel-pop-lr committed:
LPS-173575 Fix sign in error when using a new custom theme based on unstyled

ChrisKian committed:
LPS-172417 Fix test class

brianikim committed:
COMMERCE-10054 If no CommerceOrder View permission, return null and handle exception gracefully

brianikim committed:
COMMERCE-10054 Apply Exception change

Commit:
LPS-169672 Adjust Client Extensions tests

Commit:
LPS-169672 Fix message when HTML field left empty for testcase

jeffwu0724 committed:
LPS-172436 manually add the filter to get rid of \n, \r and " from fileName in petra-http-invoker

john-co committed:
LPS-169672 SF

brianikim committed:
COMMERCE-10054 Use correct groupId for front-end permissionCheck

Commit:
LPS-169905 Add class PanelCategoryRoleTypeMapperRegistry

Commit:
LPS-169905 Apply

jeffwu0724 committed:
LPS-173503 remove stream usage in KeywordQueryContributorsRegistry, also rename to avoid using stream in method name

jeffwu0724 committed:
LPS-173503 apply usage

jeffwu0724 committed:
LPS-173503 remove stream usage in ModelKeywordQueryContributorsRegistry, also rename to avoid using stream in method name

jeffwu0724 committed:
LPS-173503 apply usage

jeffwu0724 committed:
LPS-173503 remove stream usage in QueryPreFilterContributorsRegistry, also rename to avoid using stream in method name

jeffwu0724 committed:
LPS-173503 apply usage

Mikellorza committed:
LPS-172555 Add tests

Mikellorza committed:
LPS-172555 Refactor some code to use _http instead _jsonWebServiceClient.

lorenzolapescara committed:
COMMERCE-10054 Add functional tests

jeffwu0724 committed:
LPS-172436 add filter to get rid of \n, \r, and " from fileName to avoid HTTP header injection vulnerability

julschong committed:
LPS-172713 Add util-java version 2.0.0 needed classes directly and remove util-java dependency
Currently, WSDDMerger uses Java2WsddTask.generateWsdd from older util-java package. But, this class has been removed in 35ac3d8115968aa2d73ab0dc6e838bf95769aa7f. We are adding these classes directly so that this tool no longer need to reference the older util-java package.

julschong committed:
LPS-172713 Remove petra-xml dependency

brianikim committed:
COMMERCE-10054 permissionCheck for Account at the end

Mikellorza committed:
LPS-172555 Delete JSONWebServiceClient because from now on we will use Http.

JanisZhang committed:
LPS-174394 Replace with anyMatch()

kevhlee committed:
LPS-174535 Implement SF rule to check for unnecessary ConfigurationPolicy.OPTIONAL

kevhlee committed:
LPS-174535 Add unit test

JanisZhang committed:
LPS-174394 Remove Stream usages in CopyTemplateEntryMVCActionCommandTest

JanisZhang committed:
LPS-174394 Remove Stream usages in TemplateTestUtil

kevhlee committed:
LPS-174535 Apply SF

Commit:
LPS-173124 Convert method reference to field reference(The GREEDY and DYNAMIC should be able to remove as no two EntityModel servie existed at the same time, it must unregister old and re-register the new one, and class FilterParser follows class EntityModel.)

IstvanD committed:
LPS-173703 Validating the extension only if a file is being uploaded

IstvanD committed:
LPS-173703 Validating extension only if an extension is provided, and a file is uploaded, and the document type scope is not system (for example external video shortcut)

markocikos committed:
LPS-172544 Execute `onOpen` on SPA navigation within modal `iframe`

This makes `onOpen` behavior consistent with non-SPA behavior, where `onOpen` is executed after each internal navigation, as a part of `onLoad` handler. With SPA turned on, `onLoad` triggers only once.

IstvanD committed:
LPS-173703 Adding test case

julschong committed:
LPS-172713 Use SAXReaderFactory instead of new SAXReader

julschong committed:
LPS-172713 Auto, SF

tinatian committed:
LPS-173546 Remove unused methods (update) and corresonding tests

adolfopa committed:
LPS-173703 Skip validation for all system file entry types

Kvale1733 committed:
LPS-170844 Temporarily ignore failing tests

julschong committed:
LPS-174686 Remove Stream usages in AssetCategoryLayoutDisplayPageMultiSelectionProvider

dantewang committed:
LPS-174715 Update instance_wrappers.xml to build the instance wrapper directly on CalendarFactoryUtil because the interface and impl were inlined into the util in LPS-150171

dantewang committed:
LPS-174715 Run ant build-iw

robertoDiaz committed:
LPS-173884 Create the mechanism to handle the validation exception and control it

robertoDiaz committed:
LPS-173884 auto build lang

robertoDiaz committed:
LPS-173884 semver

joseabelenda committed:
LPS-174466 Implement toJSONObject in DSEnvelope

joseabelenda committed:
LPS-174466 Add tabsJSONObject in DSRecipient

ChrisKian committed:
LPS-172417 Setup the portalServletContext completely before using it to initialize the comboServlet

ChrisKian committed:
LPS-172417 Add tests

xbrianlee committed:
LPS-167306 Remove workaround

jeffwu0724 committed:
LPS-172436 run buildREST

jeffwu0724 committed:
LPS-173503 remove the usage of stream in ModelPreFilterContributorsRegistryImpl, also rename filter to filterModelPreFilterContributor in ModelPreFilterContributorsRegistry to make it clear

shuyangzhou committed:
LPS-175064 Avoid generating empty liferay-web.xml

joshchong committed:
LPS-172705 Wordsmith and sort

joshchong committed:
LPS-172705 Simplify and organize environment batch PQLs

joshchong committed:
LPS-172705 Include tests that don't explicitly set the Poshi property

joshchong committed:
LPS-172705 Run test only on centos

joshchong committed:
LPS-172705 Quarantine test from websphere tests due to bug

Commit:
LPS-169905 Semantic versioning

dantewang committed:
LPS-174686 SF, extract variable

dantewang committed:
LPS-174686 SF, use computeIfAbsent, and inline

dantewang committed:
LPS-174686 SF, flip to buy space

dantewang committed:
LPS-174686 SF, use TransformUtil and flip conditions to buy space

dantewang committed:
LPS-174686 SF, use StringUtil.split

JanisZhang committed:
LPS-174394 Simplify

dantewang committed:
LPS-173503 SF, rename

dantewang committed:
LPS-173503 SF, simplify

Commit:
LPS-174908 Replace Stream usage with JSONUtil.toJSONArray

Commit:
LPS-174910 Replace Stream usage with TransformUtil.transformToArray

haydenzheng314 committed:
LPS-174918 Remove Stream usages in MFAPolicy

Commit:
LPS-174910 Add dependencies

Commit:
LPS-174912 Remove Optional usage

markocikos committed:
LPS-172544 Simplify

markocikos committed:
LPS-172544 `onOpen` is optional

cgoncas committed:
LPS-172389 Improve readability.

Commit:
LPS-171621 Add Minifield.testcase file and stubs for FI legacy poshi tests

Commit:
LPS-171621 SF

shuyangzhou committed:
LPS-175065 Avoid bnd usage

Luizcarlosqueiroz committed:
LPS-173800 Remove not needed steps

DiegoHu97 committed:
LPS-168045 Quit alt value in Highlighted image

abhnerramos committed:
LPS-147203 Add Autom. test ClientExtension#CannotConfigureWorkflowSiteLevel

xbrianlee committed:
LPS-167306 Wait for reindex to finish for asserted asset types

abhnerramos committed:
LPS-147203 Update Autom. ClientExtension#CannotConfigureWorkflowSiteLevel

shuyangzhou committed:
LPS-174464 Simplify

kristinonias committed:
LPS-173218 Add CanTriggerActionWithExpressionByDeletingEntry test

shuyangzhou committed:
LPS-174464 Move back as inner classes.

shuyangzhou committed:
LPS-174464 Private as much as we can.

shuyangzhou committed:
LPS-174464 Remove useless protection.

joseabelenda committed:
LPS-174466 Add tabs as object in rest-openapi.yaml

shuyangzhou committed:
LPS-174464 Avoid unnecessary constructor

joseabelenda committed:
LPS-174466 Regen buildRest

joseabelenda committed:
LPS-174466 tabs as JSONObject

nNilton committed:
LPS-174946 Fix Raylife Layouts

nNilton committed:
LPS-174946 Fix Liferay-Online Layouts

nNilton committed:
LPS-174946 Fix Team Extranet Layouts

nNilton committed:
LPS-174946 Fix Jobzz Layouts

nNilton committed:
LPS-174946 Fix Masterclass Layouts

nNilton committed:
LPS-174946 Fix Testray Layouts

nNilton committed:
LPS-174946 Fix Bundle Test Layouts

Commit:
LPS-173512 Add CLAIM_MESSAGE REQUEST_ID and STATUS_BUTTON Paths

Commit:
LPS-173512 Add assertClaimMessage changeMDFStatus Macros

Commit:
LPS-173512 Add VerifyMessageWhenNoClaimWasMade Test

kevenleone committed:
LPS-175009 Update Filter and Search Schema with Fields Option

Commit:
LPS-173512 SF

kevenleone committed:
LPS-175009 Use filterSchema in ListView

abhnerramos committed:
LPS-147203 2nd Update on Autom. CannotConfigureWorkflowSiteLevel

kevenleone committed:
LPS-175009 Use filterSchema in ManagementToolbar

kevenleone committed:
LPS-175009 Simplify

kevenleone committed:
LPS-175009 Add Force Selection Option to Default Prop

abhnerramos committed:
LPS-147203 Add AssertTextPresent on CannotConfigureWorkflowSiteLevel test

marcosapmf committed:
LRAC-12924 check feature flag before returning

Commit:
LPS-175033 Add CanFilterTasksByAssignee to quarantine

kevenleone committed:
LPS-175009 Merge searchUtil into SearchBuilder

kevenleone committed:
LPS-175009 SF

tinatian committed:
LPS-173546 Baseline

shuyangzhou committed:
LPS-175065 Simplify Web-ContextPath caculating

shuyangzhou committed:
LPS-175065 Moving waiting logic to DependencyManagerSync sync callable

joshchong committed:
LPS-172705 SF

patriciajeanperez committed:
LPS-175047 Update values

joseabelenda committed:
LPS-174466 Implement toJSONObject in DSCustomField

Commit:
LPS-175054 Fix typo

dantewang committed:
LPS-172713 Enforce SAXParserFactory from JDK

izaera committed:
LPS-174464 Sort Cache-Control directives

As per https://github.com/brianchandotcom/liferay-portal/pull/129834#issuecomment-1420848369

izaera committed:
LPS-174464 Internalize resolutions hash computation in NPMRegistry

As per https://github.com/brianchandotcom/liferay-portal/pull/129875#issuecomment-1422389958

Note that we have changed NPMRegistryResolutionStateDigestUtil to receive
the list of modules and packages explicitly, instead of the NPMRegistry.
This guarantees that the digest contained in JSModulesCache will always be
coherent with the rest of the values in JSModulesCache, even if lazily
calculated.

I can't see any synchronization issue with this commit other than some
possible overhead in getResolutionStateDigest() if more than one thread
calculates the digest. But that's better than synchronizing the reads of
_resolutionStateDigest, since generally there will only be one write but
lots of reads from that variable.

cgoncas committed:
LPS-172555 Usually we name this method _setUpProperty.

izaera committed:
LPS-174464 Simplify JSResolveModulesServlet

Now that the digest is inside NPMRegistry we can simplify the servlet a
lot because we don't need to cache any information.

Note that even though the accesses to the _npmRegistry in the service()
method may look like a problem from the synchronization POV, they aren't.

This is because if the value of _npmRegistry.getResolutionStateDigest()
changes between the expectedPathInfo check and the sendRedirect() nothing
bad will happen, since the redirect will be sent using a newer digest than
the one we got when the expectedPathInfo check was made.

joseabelenda committed:
LPS-174466 baseline

cgoncas committed:
LPS-172555 We should inject the component directly using OSGi, your approach is correct for unit test, but as we are using integration tests, we can use the component and make things easier.

cgoncas committed:
LPS-172555 Better if we check only the Asah segments.

cgoncas committed:
LPS-172555 Check empty criteria too.

brianchandotcom committed:
LPS-173884 Wordsmith

brianchandotcom committed:
LPS-173884 Regen

carolmariaabb committed:
LPS-175125 Allow attachment file download on the collection display table

brianchandotcom committed:
LPS-173503 Rename

brianchandotcom committed:
LPS-173503 Rename

brianchandotcom committed:
Revert "LPS-172713 Remove petra-xml dependency"

This reverts commit 58a31b42542e947f760e01d111d7979e2d9ca5bb.

brianchandotcom committed:
Revert "LPS-172713 Remove Dom4jUtil usage in WSDDMerger and use Java2WsddTask.documentToString instead"

This reverts commit 6107563ca5e85d5fbbfcfe410f406e11756cfb2a.

brianchandotcom committed:
Revert "LPS-172713 Remove Dom4jUtil usage in Java2WsddTask and use dom4j library directly"

This reverts commit bbafeee8ba0472c01e7fdd4058f0ddcf40884077.

brianchandotcom committed:
Revert "LPS-172713 Use UnsyncStringReader instead of XMLSafeReader"

This reverts commit 9f1aef66c4b6b2ec5fabe87a0f3bc15fbf18705d.

brianchandotcom committed:
Revert "LPS-172713 Use newly added Java2WsddTask instead in WSDDBuilder"

This reverts commit c3ad55fe1407024af3df8070d15272bec3af40ff.

brianchandotcom committed:
Revert "LPS-172713 Auto, SF"

This reverts commit 130e04a7177a2cf3202949953a3d15ca720dd2a2.

brianchandotcom committed:
Revert "LPS-172713 Use SAXReaderFactory instead of new SAXReader"

This reverts commit 726cf9109b623bbf13907202d9d4ae94c9e06930.

brianchandotcom committed:
Revert "LPS-172713 Use petra-string StringPool instead"

This reverts commit 5610be2e66dba4aec1f86b83f77711504708650c.

brianchandotcom committed:
Revert "LPS-172713 Add util-java version 2.0.0 needed classes directly and remove util-java dependency"

This reverts commit fb6e0b991e11e1ab2c401d6ddff390d11d5b0823.

carolmariaabb committed:
LPS-175125 Create Util class to reuse code

carolmariaabb committed:
LPS-175125 baseline

brianchandotcom committed:
LPS-172713 prep next

brianchandotcom committed:
LPS-172713 prep next

brianchandotcom committed:
LPS-172417 Simplify and show dependency

brianchandotcom committed:
LPS-172417 Make it private

brianchandotcom committed:
LPS-172417 Assign it

brianchandotcom committed:
LPS-172417 Rename

brianchandotcom committed:
LPS-172417 Sort and put all the things with dependencies later

brianchandotcom committed:
LPS-172417 Sort

brianchandotcom committed:
LPS-172417 Why do we need this?

brianchandotcom committed:
LPS-172417 auto SF

brianchandotcom committed:
LPS-169672 Wordsmith

brianchandotcom committed:
LPS-169672 Missing a verb.

brianchandotcom committed:
LPS-169672 Wordsmith

brianchandotcom committed:
LPS-169672 Regen

brianchandotcom committed:
LPS-172713 prep next

brianchandotcom committed:
LPS-174535 prep next

julschong committed:
LPS-172719 Enforce SAXParserFactory from JDK

brianchandotcom committed:
LPS-172436 prep next

brianchandotcom committed:
LPS-172436 LPS-174535 LPS-172713 prep next

michaelhashimoto committed:
Revert "subrepo:ignore Fix sync"

This reverts commit c96e90d8f2a753386eae44c54cd81776cb10cbc3.

brianchandotcom committed:
LPS-166479 Limit for now

brianchandotcom committed:
LPS-172555 As used

brianchandotcom committed:
LPS-172555 Rename

brianchandotcom committed:
LPS-172555 Rename

brianchandotcom committed:
LPS-172555 Don't abstract it out until we need to

brianchandotcom committed:
LPS-172555 Inline/rename

brianchandotcom committed:
LPS-172555 auto SF

brianikim committed:
COMMERCE-10054 Add integration tests to reflect changes

brianchandotcom committed:
LRCI-3404 Rename to new pattern

brianchandotcom committed:
LPS-166479 Proper package

brianchandotcom committed:
LRCI-3404 new pattern

brianchandotcom committed:
LRCI-3404 stub

brianchandotcom committed:
LPS-175125 Rename

brianchandotcom committed:
LPS-172719 prep next

brianchandotcom committed:
COMMERCE-10054 sort

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
